cmd/tink-worker/cmd/container.go

```diff
@@ -0,0 +1,103 @@
+package cmd
```

Comment:
might be a good idea to move this to a new package (directory). Maybe something like `container`?

[ref](https://github.com/jacobweinstock/proposals/blob/0027-tink-worker-rearch/proposals/0027/system_architecture.png)